detekt.yaml

```diff
@@ -0,0 +1,7 @@
+exceptions:
+  TooGenericExceptionThrown:
+    active: false
+
+complexity:
+  TooManyFunctions:
+    active: false
```

Comment:
bad idea to ignore rules globally. suppress them only in places where it can be refactored